plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/HostedMachineProviderImplTest.java

```diff
@@ -114,27 +113,26 @@ public void setUp() throws Exception {
 
     @Test
     public void shouldAddMaintenanceConstraintWhenBuildImage() throws Exception {
-        provider = new HostedComposeMachineProviderImpl(dockerConnector,
-                                                        dockerConnectorConfiguration,
-                                                        credentialsReader,
-                                                        dockerMachineFactory,
-                                                        dockerInstanceStopDetector,
-                                                        containerNameGenerator,
-                                                        emptySet(),
-                                                        emptySet(),
-                                                        emptySet(),
-                                                        emptySet(),
-                                                        null,
-                                                        workspaceFolderPathProvider,
-                                                        PROJECT_FOLDER_PATH,
-                                                        false,
-                                                        false,
-                                                        emptySet(),
-                                                        emptySet(),
-                                                        SNAPSHOT_USE_REGISTRY,
-                                                        MEMORY_SWAP_MULTIPLIER,
-                                                        machineTokenRegistry,
-                                                        emptySet());
+        provider = new HostedMachineProviderImpl(dockerConnector,
```

Comment:
wdyt about creating a builder class for `HostedComposeMachineProviderImpl` for test purpose and place it near `HostedMachineProviderImplTest` which may be helpful and more clear, as it may used default values for more than a half of parameters